.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
         - npm install eslint-plugin-json --save-dev -g
         - npm install -g jshint
       ## Data model validator
-        - npm install -g fiware-model-validator
```

Comment:
- npm install -g -C ./validator

Reply:
why do you want to install this globally? I think it is clearer as it is proposed, as we are ensuring that we are running the validator locally provided in the PR.

Reply:
and we also show developers how to run the validator locally which it is rather important. 

A follow up PR could introduce npm scripts so that we just do npm test to validate the data models. that can be concurrent to the suggestion made by @jason-fox of introducing prettier in the code.